templates/zerver/slack_import.html

```diff
@@ -63,7 +63,12 @@ <h1 class="get-started">{{ _("Import from Slack") }}</h1>
                 </form>
                 {% if slack_access_token %}
                 <div class="input-box" id="slack-import-drag-drop-wrapper">
-                    <label for="slack-import-drag-and-drop" class="inline-block label-title">{{ _('Import your data') }}</label>
+                    <label for="slack-import-drag-and-drop" class="inline-block label-title">{{ _('Upload your Slack export file') }}</label>
+                    <div class="not-editable-realm-field">
+                        {% trans %}
+                        Follow <a href="https://zulip.com/help/import-from-slack#export-your-slack-data" target="_blank" rel="noopener noreferrer">these instructions</a> to obtain your Slack message history export.
+                        {% endtrans %}
```

Comment:
Doesn't `trans` support having the URL at least be specified in the `trans` tag so it's not in the translation? (which means translators need to re-translate whenever we change the URL).

We can merge this and sweep for it afterwards; this is probably not the only such bug.